polkadot-host-spec/ag-runtimeentries.tm

```diff
@@ -153,7 +164,10 @@
 
   This function should be called when a fully complete block is available
   that is not actively being built on, such as blocks received from other
-  peers.
+  peers. Any state changes created by this function on successful execution
+  are persisted. Additionally, the seal digest in the block header as
+  described in section <reference|defn-digest> must be removed manually
+  before submitting the block.
 
```

Comment:
Similarly, this should be mentioned in Chapter 3 where we define seal digest as well as in  Algorithm 3.4. Import-and-Validate-Block and Algorithm 6.5. Verify-Authorship-Right. (as the ticket suggest).

Reply:
@drskalman The removal of the seal digest before submitting to the Runtime is already mentioned in the definition of the Seal digest. Mentioning it here again  does not hurt, imo (since Soramitsu skipped over it).

I'll expand the algorithms, though.